src/modules/roc_audio/profiler.h

```diff
@@ -37,15 +37,14 @@ class Profiler : public core::NonCopyable<> {
     //! Initialization.
     Profiler(core::IAllocator& allocator,
              packet::channel_mask_t channels,
-             size_t sample_rate);
+             size_t sample_rate,
+             core::nanoseconds_t interval = LogInterval);
```

Comment:
Since it would be a pain to create a test where we are actually using 1000000000 ns, I had to keep the default value so I can override the default when testing.

Reply:
This problem appears because LogInterval is currently hardcoded in Profiler. We can just move this constant from Profiler to Pipeline (config.h). Pipeline classes will pass it to profilers they create. Tests will pass their own constants. Profiler will use this constant both for rate limiter and for calculations.